projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -102,16 +102,19 @@ const IssueScreenWithPath = ({ path }: { path: PathToIssue | undefined }) => {
                     <>
                         <Header issue={issue} />
                         <FlatList
-                            data={issue.fronts}
+                            data={issue.fronts.map((key, index) => ({
```

Comment:
Might be worth adding a clarifying comment here.

Reply:
You are very right, I meant to do that - and then didn't